src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimantResponseService.java

```diff
@@ -165,6 +177,14 @@ private AppInsightsEvent getEventNameForRejection(Claim claim) {
             : AppInsightsEvent.NON_LA_CASES;
     }
 
+    private boolean isFullAdmissionOrPartAdmission(Claim claim) {
+        Response response = claim.getResponse().orElseThrow(IllegalStateException::new);
+        ResponseType responseType = response.getResponseType();
+
+        return responseType == ResponseType.FULL_ADMISSION
```

Comment:
part paid is missing

Reply:
State paid (part) is converted to Part admission with payment declaration, so above implementation is correct.